### ISSUE TITLE
TextBook: Header image dissappears when screen is exactly 800px wide

### ISSUE THREAD

**Issue by benchilcote** [Automattic/themes#446]
## Steps to replicate

1. Activate Textbook theme and add a custom header
2. Set the browser width to exactly 800px

## Result

The header image does not show, but the space for the image remains.  At 799px, the header image and space is removed.  At 801px, the header image is displayed.

This was the case on a test site as well as a user site.  I tested the user site on Firefox and Chrome and received the same results.

Here is a GIF demonstrating my findings:

cloudup.com/cX2GMyo65zM

## Expected

I expected the header image to remain visible at 800px just like it is at 801px.

* User report: #8201594-hc
* Follow-up ticket: #1656900-zd
* User's site: https://theolounge.blog/

**Comment by laurelfulford** [comment]
I can see this too -- it looks like the header image not showing is intentional. However, the theme still shows the "desktop" header styles until a slightly narrower breakpoint, so it's a little disconcerting.

@allancole can you please confirm the above is correct (the header should not show on screens smaller than 800px)? Then we can work on getting the styles lined up to the same breakpoints.

**Comment by allancole** [comment]
@laurelfulford yup, the header image is intentionally removed on small screens, but it seems the breakpoints are off by 1 pixel.
